sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/query/OrderByUtils.java

```diff
@@ -49,7 +49,7 @@ public static <T extends Resource> Flux<OrderByRowResult<Document>> orderedMerge
                                                        targetRangeToOrderByContinuationTokenMap,
                                                        consumeComparer.getSortOrders(), clientSideRequestStatistics))
                 .toArray(Flux[]::new);
-        return Flux.mergeOrdered(consumeComparer, fluxes);
+        return Flux.mergeOrdered(1, consumeComparer, fluxes);
```

Comment:
Worth adding a comment here to explain why we are prefetching just 1. Similar to the comment we added on the flatMap prefetch fix PR.

Reply:
Added the comment